src/Messenger/IndexationRequest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace JoliCode\Elastically\Messenger;
+
+final class IndexationRequest
+{
+    private $op;
+    private $type;
+    private $id;
+
+    public function __construct(string $type, string $id, string $op = IndexationRequestHandler::OP_INDEX)
```

Comment:
Why do you call `type` like that? It's actually an index, isn't ?
CF: the doc:
```php
// New Client object with new options
$client = new Client([
    Client::CONFIG_INDEX_CLASS_MAPPING => [
        'beers' => Beer::class,    
    ],
])
```

Reply:
It's the class name we expect here, not the index name.

Reply:
so classname is better? As you may see, I didn't understand this code :/